src/components/site-header/styles/_Header.scss

```diff
@@ -20,8 +20,11 @@ $getsmarter-logo-bg: #082135;
       max-height: 50px;
   }
 
-  .getsmarter-logo {
-    background-color: $getsmarter-logo-bg;
+  .logo-right {
+    border-left: 1px solid lightgrey;
```

Comment:
Rather than exposing `lightgrey` via SCSS exports, use the `$gray-200` SCSS variable here directly, i.e. `border-left: 1px solid $gray-200;`